src/k2/persistence/logStream/LogStream.h

```diff
@@ -0,0 +1,175 @@
+/*
+MIT License
+
+Copyright(c) 2020 Futurewei Cloud
+
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions :
+
+    The above copyright notice and this permission notice shall be included in all copies
+    or
+    substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS",
+    WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+    DAMAGES OR OTHER
+    LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+*/
+
+#pragma once
+
+#include <k2/transport/PayloadSerialization.h>
+#include <seastar/core/sharded.hh>
+#include <k2/transport/Payload.h>
+#include <k2/transport/Status.h>
+#include <k2/common/Common.h>
+#include <k2/config/Config.h>
+#include <k2/cpo/client/CPOClient.h>
+#include <k2/transport/BaseTypes.h>
+#include <k2/transport/TXEndpoint.h>
+#include <k2/persistence/plog_client/PlogClient.h>
+
+
+namespace k2 {
+
+namespace log {
+inline thread_local k2::logging::Logger lgbase("k2::logstream_base");
+}
+
+enum LogStreamType : Verb {
```

Comment:
We share the verb definitions across the board. THese are in dto/MessageVerbs.h . You should move this there and pick a range that doesn't conflict

Reply:
changed all the verb to K2_DEF_ENUM